latch_cli/services/get_params.py

```diff
@@ -84,83 +66,132 @@ def get_params(wf_name: str, wf_version: Optional[str] = None):
     default_wf_vars = wf_default_params["parameters"]
     for key, value in wf_vars.items():
         try:
-            description_json = json.loads(value["description"])
+            description_json_str = value["description"]
+            literal_type_json = value["type"]
+        except KeyError as e:
+            raise ValueError(
+                f"Flyte workflow interface for "
+                f"{wf_name}-{wf_version} is missing 'description' or "
+                "'type' key"
+            ) from e
+
+        try:
+            description_json = json.loads(description_json_str)
+        except json.decoder.JSONDecodeError:
+            # Parameters that are used for control flow in forks do not have
+            # valid JSON in description and we can safely ignore these.
+            # TODO - add metadata to control flow fork parameters to exclude
+            # these specifically.
+            continue
+
+        try:
             param_name = description_json["name"]
-        except (json.decoder.JSONDecodeError, KeyError) as e:
+        except KeyError as e:
             raise ValueError(
-                f"Parameter description json for workflow {wf_name} is malformed"
+                f"Parameter description json for workflow {wf_name} and parameter {key} is missing 'name' key."
             ) from e
 
-        literal_type_json = value["type"]
         literal_type = gpjson.ParseDict(literal_type_json, _LiteralType())
 
-        python_type = _guess_python_type(
-            LiteralType.from_flyte_idl(literal_type), param_name
-        )
+        python_type = guess_python_type(LiteralType.from_flyte_idl(literal_type))
 
         default = True
         if default_wf_vars[param_name].get("required") is not True:
             literal_json = default_wf_vars[param_name].get("default")
             literal = gpjson.ParseDict(literal_json, _Literal())
-            val = _guess_python_val(Literal.from_flyte_idl(literal), python_type)
+            val = guess_python_val(Literal.from_flyte_idl(literal), python_type)
         else:
             default = False
-            val = _best_effort_default_val(python_type)
+
+            val = best_effort_python_val(python_type)
 
         params[param_name] = (python_type, val, default)
 
-    import_statements = {
-        LatchFile: "from latch.types import LatchFile",
-        LatchDir: "from latch.types import LatchDir",
-        enum.Enum: "from enum import Enum",
+    import_statements: typing.Dict[str, typing.List[str]] = {
```

Comment:
just import all of these by default, theres no real need to dynamically add them and make this function more complicated

Reply:
would rather have correct imports than all imports to avoid confusing user with potentially long preamble of unnecessary statements

Reply:
so unless have ideas of how to do simpler dont want to remove